android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -376,10 +376,17 @@ FUNC(jstring, resultToString, jint result) {
 FUNC(jstring, versionString) {
   char codec_versions[256];
   avifCodecVersions(codec_versions);
+  char libyuv_version[64];
+  if (avifLibYUVVersion() > 0) {
+    snprintf(libyuv_version, sizeof(libyuv_version), " libyuv: %d.",
```

Comment:
Nit: Use `%u` instead of `%d` because `avifLibYUVVersion()` returns `unsigned int`.

Reply:
https://github.com/AOMediaCodec/libavif/pull/1614